resources/cluster-essentials/templates/crd-init-api-gateway.yaml

```diff
@@ -17,14 +17,36 @@ spec:
       serviceAccountName: {{ .Release.Name }}-crd-init
       containers:
       - name: {{ .Release.Name }}-crd-apirules
-        image: "{{ .Values.rbacJob.image.repository }}:{{ .Values.rbacJob.image.tag }}"
+        image: "{{ .Values.jobs.image.repository }}:{{ .Values.jobs.image.tag }}"
         volumeMounts:
         - name: crd-apirules
           mountPath: /etc/crd
           readOnly: true
-        command: ["kubectl",  "apply", "-f", "/etc/crd/apirules.yaml"]
+        command:
```

Comment:
Since we are doing bash in bash here, maybe to simplify this structure a little bit, move it to a separate file in the files directory, and include it with Files.Get? The current form is very hard to read

Reply:
[please check this comment](https://github.com/kyma-project/kyma/pull/8370/files#r431739607)